src/containers/ProfileDetails/components/PasspointProfile/index.js

```diff
@@ -255,25 +253,13 @@ const PasspointProfileForm = ({
     {
       title: 'Radio',
       dataIndex: ['details', 'appliedRadios'],
-      render: appliedRadios => appliedRadios?.map(i => radioTypes?.[i])?.join(',  '),
-    },
-    {
-      title: '',
-      width: 80,
-      render: (_, record) => (
-        <Button
-          title="removeSsid"
-          icon={<DeleteOutlined />}
-          onClick={() => handleRemoveSsid(record.id)}
-        />
-      ),
+      render: appliedRadios =>
+        sortRadioTypes([...appliedRadios])
```

Comment:
Isnt `[...appliedRadios]` the same as jsut `appliedRadios` ?

Reply:
Yes. But the `appliedRadios` array was frozen in [strict-mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) so  I needed to copy the array before sorting it

Reply:
True